structured_prediction_baselines/models/base.py

```diff
@@ -92,7 +92,10 @@ def get_extra_args_for_loss(
         labels: torch.Tensor,
         **kwargs: Any,
     ) -> Dict:
-        return {}
+        mask = util.get_text_field_mask(x)
```

Comment:
Calculation of the mask should not be in the base model. This will break the multilabel model! This function should be overridden in the sequence tagging model only.